### DISCUSSION TITLE
question about async_hooks

### DISCUSSION THREAD

**Discussion by theanarkh** [nodejs/help#4715] [Q&A]
Hi, all! I wonder if this document is wrong ?  I think the return value of `async_hooks.triggerAsyncId();` is `bootstrap`.
<img width="913" alt="image" src="https://user-images.githubusercontent.com/21155906/233332268-4e0e1c70-4ae3-42d9-9418-4e0741899b3c.png">

**Comment by VoltrexKeyva** [comment]
The documentation is correct, `async_hooks.triggerAsyncId()` returns a number.

**Comment by theanarkh** [comment]
Sorry, i mean the return value of async_hooks.triggerAsyncId() is the asyncId of bootstrap instead of the asyncId of the new connection.
